python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -65,48 +69,65 @@ def setDefaults(self):
 ## @}
 
 class ProcessImageForcedTask(lsst.pipe.base.CmdLineTask):
-    """!
-    A base class for command-line forced measurement drivers.
+    """!A base class for command-line forced measurement drivers.
 
     This is a an abstract class, which is the common ancestor for ForcedPhotCcdTask
     and ForcedPhotCoaddTask.  It provides the run() method that does most of the
     work, while delegating a few customization tasks to other methods that are
     overridden by subclasses.
+
+    This task is not directly usable as a CmdLineTask; subclasses must:
+     - Set the _DefaultName class attribute
+     - Implement makeIdFactory
+     - Implement fetchReferences
+     - (optional) Implement attachFootprints
     """
     ConfigClass = ProcessImageForcedConfig
     _DefaultName = "processImageForcedTask"
 
     def __init__(self, butler=None, refSchema=None, **kwds):
+        """Initialize the task.
+
+        ForcedPhotImageTask takes two keyword arguments beyond the usual CmdLineTask arguments:
+         - schema: the Schema of the reference catalog, passed to the constructor of the references subtask
+         - butler: a butler that will be passed to the references subtask to allow it to load its Schema
+           from disk.
+        At least one of these arguments must be present; if both are, schema takes precedence.
+        """
```

Comment:
Actual function argument is `refSchema`, not `schema`; documentation should be changed to reflect this.